server/src/connectionHandlers/onCompletion.ts

```diff
@@ -0,0 +1,62 @@
+import logger from 'winston'
+import { type TextDocumentPositionParams, type CompletionItem, type SymbolInformation, CompletionItemKind } from 'vscode-languageserver/node'
+import { symbolKindToCompletionKind } from '../utils/lsp'
+import { RESERVED_VARIABLES } from '../completions/reserved-variables'
+import { RESERVED_KEYWORDS } from '../completions/reserved-keywords'
+import { analyzer } from '../tree-sitter/analyzer'
+import { SNIPPETS } from '../completions/snippets'
+
+export function onCompletionHandler (textDocumentPositionParams: TextDocumentPositionParams): CompletionItem[] {
+  logger.debug('onCompletion')
+  //   const documentAsText = documentAsTextMap.get(textDocumentPositionParams.textDocument.uri)
+  //   if (documentAsText === undefined) {
```

Comment:
dead code. are you sure you want to keep it?

Reply:
hmm, there shouldn't be a problem to remove it because the code below is already doing a similar thing.